image/scanner/rhel/konflux.Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Compiling scanner binaries and staging repo2cpe and genesis manifests
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.23@sha256:0a070e4a8f2698b6aba3630a49eb995ff1b0a182d0c5fa264888acf9d535f384 AS builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.24@sha256:021ab8e6e4af3d1683b52ec985996f9421edb2111dc697b5781dee3c7595ae00 AS builder
```

Comment:
I added `konflux-build` label because the builds don't trigger in PRs without it (we can revisit).
Please push any/empty commit to this PR to trigger Konflux CI.
Let's make sure `openshift-golang-builder` is also happy with the change.

Reply:
Thanks! I forgot the Konflux builds didn't run by default

Reply:
It looks like they changed the name of the tag, so I had to update that, too. I chose the option I figured would be best